tests/test_flask.py

```diff
@@ -78,25 +79,27 @@ def test_endpoint(var1: str, var2: str) -> dict[str, str]:
 
     dbos._sys_db.wait_for_buffer_flush()
     # Change the workflow status to pending
-    dbos._sys_db.update_workflow_status(
-        {
-            "workflow_uuid": wfuuid,
-            "status": "PENDING",
-            "name": test_workflow.__qualname__,
-            "class_name": None,
-            "config_name": None,
-            "output": None,
-            "error": None,
-            "executor_id": None,
-            "app_id": None,
-            "app_version": None,
-            "request": None,
-            "recovery_attempts": None,
-            "authenticated_user": None,
-            "authenticated_roles": None,
-            "assumed_role": None,
-            "queue_name": None,
-        }
+    (
```

Comment:
Wow this kind of noise is really unfriendly to reviewers.  I guess it's the same, just extra parens?

Reply:
Sorry about that. Where there was only an async version of `update_workflow_status`, the call was wrapped in a call to `asyncio.run`. I removed the `asyncio.run` call and must have forgotten to remove parens.